content/en/docs/concepts/signals/logs.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Logs
 weight: 3
+description: A timestamped text record
```

Comment:
May not be strictly "text" (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#type-any).

Reply:
Again, this is wording copied from the concept page about logs: https://opentelemetry.io/docs/concepts/signals/logs/, I am also not 100% happy with the wording so suggestions are welcomed (`A timestamped record` seems odd)

Reply:
I removed that `description` for now. This creates inconsistency but as of now I don't have a better wording.